app/views/kaui/payments/_payment_table.html.erb

```diff
@@ -96,8 +96,20 @@
             <td><%= transaction.transaction_type %></td>
             <td>
               <% if transaction.amount.present? %>
+                  <% if transaction.amount != transaction.processed_amount %>
+                  <%= humanized_money_with_symbol Kaui::Transaction.processed_amount_to_money(transaction) %>
+                  (<%= transaction.currency %>)
+                  <a href="#" data-toggle="tooltip" class="kb-tooltip" title="<%= humanized_money_with_symbol Kaui::Transaction.amount_to_money(transaction) %>(<%= transaction.currency %>)">(?)</a>
+                  <div class="tooltip bs-tooltip-top" role="tooltip">
+                    <div class="arrow"></div>
+                    <div class="tooltip-inner">
+                      <%= humanized_money_with_symbol Kaui::Transaction.amount_to_money(transaction) %>(<%= transaction.currency %>)
```

Comment:
Maybe we should show a legend, to make it clear, e.g. `Requested amount: <%= humanized_money_with_symbol Kaui::Transaction.amount_to_money(transaction) %> (<%= transaction.currency %>)`?

Reply:
![Screenshot 2022-10-14 at 11 45 52](https://user-images.githubusercontent.com/65641209/195816970-d2ed70a9-7adf-40c1-8722-60f5b8970227.png)